pkg/ddc/jindofsx/transform.go

```diff
@@ -159,11 +160,11 @@ func (e *JindoFSxEngine) transform(runtime *datav1alpha1.JindoRuntime) (value *J
 	e.transformNetworkMode(runtime, value)
 	e.transformFuseNodeSelector(runtime, value)
 	e.transformSecret(runtime, value)
-	e.transformToken(runtime, value)
-	err = e.transformMaster(runtime, metaPath, value, dataset)
+	err = e.transformMaster(runtime, metaPath, value, dataset, smartdataTag, fuseTag)
```

Comment:
Can we add if else logic with two method transformMaster and transformMasterForDeprecatedVersion?